### PR TITLE
Update doc building instructions

### PR DESCRIPTION
**Pull request by czerouni**
These changes just add detail to the instructions for setting up and building the documentation.

### CODE REVIEW

File: docs/conf.py
Comment (doug-walker) [inline docs/conf.py:133]: Unintentional commented line.

**Review by doug-walker — APPROVED** [review]
Thanks for these fixes Craig!

File: docs/conf.py
Comment (czerouni) [reply docs/conf.py:133]: Should I remove that line?

Comment (doug-walker) [reply docs/conf.py:133]: Yes, please. GitHub is showing that you added it. It doesn't seem to serve a purpose.

Comment (czerouni) [reply docs/conf.py]: Done

Comment (doug-walker) [reply docs/conf.py]: Thanks Craig.  We are waiting on the second approval and then should be able to merge.

**Review by zachlewis — APPROVED** [review]

**Pull request merged**
